tests/test_generic.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+from pyro.generic import infer, pyro_backend, handlers
+from pyro.generic.testing import MODELS
+
+
+pytestmark = pytest.mark.stage('unit')
+
+
+@pytest.mark.parametrize('model', MODELS)
+@pytest.mark.parametrize('backend', ['pyro'])
```

Comment:
Can we add the `minipyro` backend here, or does that not support MCMC?

Reply:
`minipyro` doesn't support MCMC. Maybe we can add a vanilla HMC to minipyro that uses the integrator in `pyro.ops`, but I'm not sure if it will be as concise as SVI. It could be a nice little hackathon project.